### PR TITLE
conduit support for visit

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

File: var/spack/repos/builtin/packages/visit/package.py [hunk 1]
```diff
@@ -227,6 +228,13 @@ class Visit(CMakePackage):
     depends_on('silo+mpi', when='+silo+mpi')
     depends_on('silo~mpi', when='+silo~mpi')
 
+    depends_on('conduit@0.8.3', when='+conduit')
```
Comment (chuckatkins) [inline var/spack/repos/builtin/packages/visit/package.py]: Is `@0.8.3` a hard requirement or can we use `@0.8:`?

Also is there a version constraint on visit to support conduit?

Comment (cyrush) [reply var/spack/repos/builtin/packages/visit/package.py]: Sorry I intended to do `0.8.3:` -- does that sound better?

I think `0.8.2` has a build issue that we could hit. 

No version constraint for VisIt, conduit support has been there for quite some time.

**Review by chuckatkins — APPROVED** [review]

**Comment by cyrush** [comment]
gitlab: looks like job logs hit max size before whatever the error is was captured.

**Comment by cyrush** [comment]
Block is VisIt  is failing to detect MPI at configure. 

Not sure what to make of that, since adding Conduit should not cause changes to MPI in VisIt, but maybe it results in some spec change to MPI.

```
-- Looking for SILO
--   Found library siloh5 in /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spa/linux-ubuntu18.04-x86_64/gcc-7.5.0/silo-4.10.2-4gxqfnfpidrgxrbedehzpo6jbm5c62t2/lib
--   SILO found
--     Using PDB Lite built into Silo
-- Looking for XDMF
--   XDMF not requested
-- Looking for MDSPLUS
--   MDSPLUS not requested
-- Looking for UINTAH
--   UINTAH not requested
-- UINTAH_INTERFACES_LIB = NOTFOUND
-- PIDX library not found. Try setting PIDX_DIR
-- Looking for GFORTRAN
--   GFORTRAN not requested
-- Setting up MPI using compiler wrapper
-- Could NOT find MPI_C (missing: MPI_C_WORKS) 
-- Could NOT find MPI_CXX (missing: MPI_CXX_WORKS) 
-- Could NOT find MPI (missing: MPI_C_FOUND MPI_CXX_FOUND) 
CMake Error at CMakeLists.txt:1449 (MESSAGE):
  Failed to setup MPI using compiler wrapper:
  /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spa/linux-ubuntu18.04-x86_64/gcc-7.5.0/openmpi-4.1.3-c6u2ejhxt2o2hac7mtzlgtffxsf5iv5d/bin/mpic++
Call Stack (most recent call first):
  CMakeLists.txt:1490 (DETECT_MPI_SETTINGS)


```

**Comment by alalazo** [comment]
@spackbot run pipeline

**Comment by alalazo** [comment]
@cyrush I couldn't retrigger pipelines myself, can you please rebase this PR?

**Comment by cyrush** [comment]
Closing in favor of already merged https://github.com/spack/spack/pull/32255

**Pull request closed without merging**